src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -6188,7 +6578,12 @@ namespace HVACVariableRefrigerantFlow {
         }
 
         SetAverageAirFlow(VRFTUNum, 0.0, OnOffAirFlowRatio);
-    } // namespace HVACVariableRefrigerantFlow
+    
+        if (ErrorsFound) {
```

Comment:
Noticed to existing warnings that were previously moved to Init never had a fatal applied if ErrorsFound was set to true.